src/Tizen.Security.DevicePolicyManager/Tizen.Security.DevicePolicyManager/PolicyState.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ *  Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ */
+
+using System;
+
+namespace Tizen.Security.DevicePolicyManager
+{
+    /// <summary>
+    /// Enumeration for the device policy state.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public enum PolicyState
+    {
+        /// <summary>
+        /// The policy status is disallowed.
+        /// </summary>
+        DISALLOWED = 0,
+        /// <summary>
+        /// The policy status is allowed.
+        /// </summary>
+        ALLOWED,
+        /// <summary>
+        /// The policy status is unknown.
+        /// </summary>
+        UNKNOWN
```

Comment:
Is it really need? why not simply use Disallowed?
`Unknow` can be returned in normal state?

Reply:
I remove "Unknown". and I added comment default value of properties.